oat-sa/tao-core-ui-fe
# [ACTP-311] Login Page - Fix Login Component and Focus Style

Related to : https://oat-sa.atlassian.net/browse/ACTP-311
 
Refactor login form:

- get rid of labels duplications between real and fake form,  when autocomplete is disabled
-  make autofocus on login field configurable
-  add aria attributes to reveal password button
 
#### How to test
 
- Prepare an instance of TAO with taoAct extension
- Navigate to login page
- Check that there is no autofocus on login field and screen reader reads page from the top
- Check that screen reader can recognize show password controls
- check focus styles of `login` and guest `access buttons`
   
Companion PR :
 - [ ] https://github.com/oat-sa/tao-core/pull/2406
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1308
